packages/miniflare/src/workers/kv/assets.worker.ts

```diff
@@ -0,0 +1,34 @@
+import { SharedBindings } from "miniflare:shared";
+import { decodeAssetsKey, KVParams } from "./constants";
+
+interface Env {
+	[SharedBindings.MAYBE_SERVICE_BLOBS]: Fetcher;
+}
+
+export default <ExportedHandler<Env>>{
+	async fetch(request, env) {
+		// Only permit reads
+		if (request.method !== "GET") {
+			const message = `Cannot ${request.method.toLowerCase()}() with Workers Assets namespace`;
+			return new Response(message, { status: 405, statusText: message });
+		}
+
+		// Decode key
+		const url = new URL(request.url);
+		let key = url.pathname.substring(1); // Strip leading "/"
+		if (url.searchParams.get(KVParams.URL_ENCODED)?.toLowerCase() === "true") {
+			key = decodeURIComponent(key);
+		}
+
+		// Strip ASSETS_NO_CACHE_PREFIX
+		key = decodeAssetsKey(key);
```

Comment:
Remove?
Or not if we feel that we should leave this cache busting in place

Reply:
✅